keyboards/ploopyco/ploopyco.c

```diff
@@ -158,15 +181,11 @@ bool process_record_kb(uint16_t keycode, keyrecord_t* record) {
 
     if (keycode == DRAG_SCROLL) {
 #ifndef PLOOPY_DRAGSCROLL_MOMENTARY
-        if (record->event.pressed)
-#endif
-        {
+        is_drag_scroll = record->event.pressed;
+#else
+        if (record->event.pressed) {
             is_drag_scroll ^= 1;
         }
```

Comment:
I believe this condition to be inverted. When I `#define PLOOPY_DRAGSCROLL_MOMENTARY` in my keymap config `DRAG_SCROLL`  behaves as as toggled. 

When I don't define `PLOOPY_DRAGSCROLL_MOMENTARY` it behaves as momentary

Reply:
whoops, you're right